### DISCUSSION TITLE
Custom tenant menu

### DISCUSSION THREAD

**Discussion by piotrh052b** [filamentphp/filament#9346] [Help]
Hello, how can I replace or remove tenant menu component from sidebar ?

I have a lots of tenants and I would like to replace standard tenant menu with searchable select, or at least replace it with button to call modal with scrollable list.

Is this possible ?

**Comment by suleymanozev** [comment]
Did you find a solution?

**Reply by piotrh052b** [reply]
Unfortunately not, but there was already a similar thread: [7872](https://github.com/filamentphp/filament/discussions/7872)

**Comment by SamAsEnd** [comment]
Any solution yet?

**Reply by devhoussam1998** [reply]
https://filamentphp.com/docs/3.x/panels/tenancy#hiding-the-tenant-menu

**Comment by d8vjork** [comment]
Searching for this already found something (don't know if it was coming from Filament v4 or 4.1), but is actually working.

Adding this to your panel provider:

```php
->searchableTenantMenu()
```
